### PR TITLE
[LSD-203] Adding CO2 Vent to set flags on the definitions

### PR DESCRIPTION
**Pull request by gausnes**
\cc @workingmonk @Dianoga

### CODE REVIEW

File: smartapps/dianoga/co2-vent/src/co2-vent.groovy [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        description: "Turn on a switch when CO2 levels are too high",
+        category: "Health & Wellness",
+        iconUrl: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience.png",
+        iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience@2x.png",
+        pausable: true
+)
+
+preferences {
+    section("CO2 Sensor") {
+        input "sensor", "device.netatmoBasestation", title: "Sensor", required: true
```
Comment (workingmonk) [inline smartapps/dianoga/co2-vent/src/co2-vent.groovy]: @Dianoga any reason why this using netatmo instead of the capability?

**Comment by Dianoga** [comment]
When I wrote it, there was no capability for CO2

**Comment by workingmonk** [comment]
Cool. in that case do you want to fix this now before we merge it in?

**Comment by gausnes** [comment]
@workingmonk updated to `capability.carbonMonoxideDetector`

**Comment by workingmonk** [comment]
@gausnes i think you used the monoxide detector instead of dioxide detector

**Comment by gausnes** [comment]
@workingmonk fixed. It feels kind of weird that is it `carbonDioxideMeasurement` instead of `carbonDioxideDetector`

**Comment by workingmonk** [comment]
I know. I don't understand it either. @Dianoga was the intent of the app to measure CO instead of CO2?

**Comment by Dianoga** [comment]
No. The intention of this app is to monitor carbon _dioxide_. This app along with the Netatmo sensor it originally referred to were created before there was much in the way of available capabilities.

**Comment by workingmonk** [comment]
just realized that the file path is incorrect.
`smartapps/dianoga/co2-vent/src/co2-vent.groovy` should be
`smartapps/dianoga/co2-vent.src/co2-vent.groovy`

**Comment by workingmonk** [comment]
looks good after that to merge

**Comment by gausnes** [comment]
Intellji was trying to be fancy 😛 fixed.

**Review by workingmonk — APPROVED** [review]

**Pull request merged**